config/Version.public.xcconfig

```diff
@@ -1,5 +1,5 @@
-VERSION_SHORT=1.12.0
+VERSION_SHORT=1.12
 
 // Public long version example: 1_12_0_1. The last 1 means the first build for
 // this version
-VERSION_LONG=11202
+VERSION_LONG=1.12.0.2
```

Comment:
I worry this change won't go well with Apple 😞 A way to test this would be to ship a new build to TestFlight.

In https://github.com/Automattic/simplenote-macos/commit/f174a5b4afd66fc59f983448780af788e9808f10 I reverted a similar change because App Store Connect rejected the build with message

> This bundle is invalid. The value for key CFBundleVersion
> [1.11.1.0] in the Info.plist file must contain a higher version than
> that of the previously uploaded version [11103].

I wonder if we may need to bump the build number even more, i.e. adding another leading 1? But maybe that wouldn't work with GlotPress?

> This was causing issues when updating the Metadata GlotPress Key.

It might be that we need to do some extra work on the plugin side.

Reply:
@jleandroperez 

> I haven't found where is it, exactly, that VERSION_LONG is updated. Is that a manual step?

Yes, it's a manual step right now 😳